app/controllers/support_interface/change_name_controller.rb

```diff
@@ -0,0 +1,47 @@
+module SupportInterface
+  class ChangeNameController < SupportInterfaceController
+    include Pagy::Backend
+    include ConsumesIdentityUsersApi
+
+    layout "two_thirds"
+
+    def show
+      @user = identity_users_api.get_user(uuid)
+      @change_name_form = ChangeNameForm.new
+      @uuid = uuid
+    end
+
+    def update
+      @user = identity_users_api.get_user(uuid)
+      @change_name_form = ChangeNameForm.new(change_name_params)
+      @uuid = uuid
+      if @change_name_form.save
+        @user =
+          identity_users_api.update_user(
```

Comment:
Would this be better placed in the body of the `save` method ?

Reply:
Hmm that's also a possibility, I think I like keeping request-y stuff in the controller; if it was a save to a local database I would have probably done it here